_data/projects/faas.yml

```
@@ -0,0 +1,15 @@
+---
+name: faas
+desc: OpenFaaS - Serverless Functions Made Simple
```

If you prefer the name above, this can be simplified:

```suggestion
desc: Serverless Functions Made Simple
```